pkg/hostagent/hostagent.go

```diff
@@ -317,6 +317,18 @@ func (a *HostAgent) startHostAgentRoutines(ctx context.Context) error {
 	if err := a.waitForRequirements(ctx, "essential", a.essentialRequirements()); err != nil {
 		mErr = multierror.Append(mErr, err)
 	}
+	if *a.y.SSH.ForwardAgent {
+		faScript := `#!/bin/bash
+set -eux -o pipefail
+sudo mkdir -p -m 755 /run/host-services
```

Comment:
Should this be chowned to the main user with chmod 0700 ?

Reply:
I don't know; as long as the content are just symlinks, the real access control is done by the permission of the symlink targets. The symlinks themselves are just labels.

Reply:
We could...

```
sudo mkdir -p -m 700 /run/host-services
sudo ln -sf "${SSH_AUTH_SOCK}" /run/host-services/ssh-auth.sock
sudo chown -R "${USER}" /run/host-services
```

Reply:
We could indeed. I don't see the point, but it doesn't hurt either, so 🤷

Reply:
Done.